docs/provider/doppler.md

```diff
@@ -30,7 +36,30 @@ Then to create a generic `SecretStore`:
 {% include 'doppler-generic-secret-store.yaml' %}
 ```
 
-> **NOTE:** In case of a `ClusterSecretStore`, be sure to set `namespace` in `secretRef.dopplerToken`.
+### OIDC Authentication
+
+For OIDC authentication, you'll need to configure a Doppler [Service Account Identity](https://docs.doppler.com/docs/service-account-identities) and create a Kubernetes ServiceAccount.
+
+First, create a Kubernetes ServiceAccount:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: doppler-oidc-sa
+  namespace: external-secrets
+```
+
+Next, create a Doppler Service Account Identity with:
+- **Issuer**: Your cluster's OIDC discovery URL
+- **Audience**: The resource-specific audience for the SecretStore (e.g., `secretStore:external-secrets:doppler-auth-api` for a SecretStore or `clusterSecretStore:doppler-auth-api` for a ClusterSecretStore)
+- **Subject**: The Kubernetes ServiceAccount (e.g., `system:serviceaccount:external-secrets:doppler-oidc-sa`)
```

Comment:
Could we explain this in terms of a template as well as provide concrete examples? (i.e., `secretStore:<namespace>:<secretStoreName>` and `system:serviceaccount:<namespace>:<serviceAccountName>`)